src/tests/multihost/admultidomain/test_multidomain.py

```diff
@@ -193,3 +194,81 @@ def test_0003_bz2167728(multihost, newhostname, adchildjoin):
         assert getent1.returncode == 0, f'Could not find user1@{domain}!'
         assert getent2.returncode == 0, f'Could not find child_user1@{child_domain}!'
         assert getent3.returncode == 0, f'Could not find tree_user1@{tree_domain}!'
+
+    @pytest.mark.ticket(bz=1913284, jira=["SSSD-3092", "RHEL-4974"])
+    @staticmethod
+    def test_0004_bz1913284_keytab_as_nonroot(multihost, newhostname, adchildjoin):
+        """
+        :title: IDM-SSSD-TC: ad_provider: forests: krb5_kt_start_seq_get failed:
+          Permission denied when running as unprivileged user sssd
+        :id: 53a6871e-95a6-4865-9e61-1e12815ec35b
+        :setup:
+          1. Configure parent and child domain
+          2. Join client to child domain
+        :steps:
+          1. Lookup user from child domain
+          2. Lookup user from parent domain
+          3. Check log for a keytab error
+        :expectedresults:
+          1. Parent user is found
+          2. Child user is found
+          3. The permission denied error is not present in log.
+        :customerscenario: True
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1913284
+        """
+        adchildjoin(membersw='adcli')
+        ad_domain = multihost.ad[0].domainname
+        child_domain = multihost.ad[1].domainname
+        ad_server = multihost.ad[1].hostname
+
+        # Configure sssd
+        multihost.client[0].service_sssd('stop')
+        client = sssdTools(multihost.client[0], multihost.ad[1])
+        client.backup_sssd_conf()
+        dom_section = f'domain/{client.get_domain_section_name()}'
+        sssd_params = {
+            'ad_domain': child_domain,
+            'debug_level': '9',
+            'use_fully_qualified_names': 'True',
+            'ad_server': ad_server,
+            'cache_credentials': 'True',
+        }
+        client.sssd_conf(dom_section, sssd_params)
+        # Set sssd to run as user sssd
+        if client.sssd_user != "sssd":
+            with tempfile.NamedTemporaryFile(mode='w') as tfile:
+                tfile.write("[sssd]\nuser = sssd\n")
```

Comment:
This legacy method doesn't work anymore (its support isn't even built by default).
This PR is blocked by #7266

Reply:
I created the test code draft before we changed to non-root by default.
I kept this piece around if the fix is back-ported to 2.9 and we need to run it there.

Reply:
> if the fix is back-ported to 2.9

There is no way to backport a fix for this issue only without all other "unprivileged" patches.
So this is 2.10+ only

Reply:
Removed.